samtranslator/open_api/open_api.py

```diff
@@ -415,7 +415,8 @@ def _set_method_authorizer(self, path, method_name, authorizer_name, authorizers
             security_dict = dict()
             security_dict[authorizer_name] = []
 
-            if authorizer_name != "NONE":
+            # Neither the NONE nor the AWS_IAM built-in authorizers support authorization scopes.
+            if authorizer_name != "NONE" and authorizer_name != "AWS_IAM":
```

Comment:
nit: You can use `not in ["NONE", "AWS_IAM"]` here

Reply:
Fancy; seems like a reasonable change.